File: branches/Medusa/Source/Fantasy.Jobs/Resources/JobMetaDataFilter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Fantasy.Jobs.Management;
using Fantasy.Jobs;

namespace Fantasy.Jobs.Temporary
{
    public class JobMetaDataFilter : IJobMetaDataFilter
    {

        #region IJobMetaDataFilter Members

        public IEnumerable<JobMetaData> Filter(IQueryable<JobMetaData> source)
        {
            <%=vars%>

            var rs = source;

            
            <%=enableCondition%>rs = from job in rs where  <%=condition%> select job;

            

            <%=enableOrder%>rs = from job in rs orderby <%=order%> select job ; 
            
           
           return rs;
        }

        #endregion
    }
}

File: AutoConverted/Fantasy.Jobs/Resources/JobMetaDataFilter.java
package Fantasy.Jobs.Temporary;

import Fantasy.Jobs.Management.*;
import Fantasy.Jobs.*;

public class JobMetaDataFilter implements IJobMetaDataFilter
{

//C# TO JAVA CONVERTER TODO TASK: There is no preprocessor in Java:
		///#region IJobMetaDataFilter Members

	public final Iterable<JobMetaData> Filter(IQueryable<JobMetaData> source)
	{
		<%=vars%> var rs = source;


//C# TO JAVA CONVERTER TODO TASK: There is no Java equivalent to LINQ queries:
		<%=enableCondition%>rs = from job in rs where <%=condition%> select job;



//C# TO JAVA CONVERTER TODO TASK: There is no Java equivalent to LINQ queries:
		<%=enableOrder%>rs = from job in rs orderby <%=order%> select job;


	   return rs;
	}

//C# TO JAVA CONVERTER TODO TASK: There is no preprocessor in Java:
		///#endregion
}